es_ES/faq.md

```diff
@@ -0,0 +1,625 @@
+[StatelessWidget](https://docs.flutter.io/flutter/widgets/StatelessWidget-class.html), que son esencialmente funciones que describen cómo los argumentos se asignan a otras funciones, cayendo en primitivas que computan layouts o gráficos de pintura. (Tales aplicaciones no pueden tener estado fácilmente, por lo que normalmente no son interactivas.)
+Por ejemplo, el [Icon](https://docs.flutter.io/flutter/widgets/Icon-class.html)
+widget es esencialmente una función que mapea sus argumentos
+([color](https://docs.flutter.io/flutter/widgets/Icon/color.html),
+[icon](https://docs.flutter.io/flutter/widgets/Icon/icon.html),
+[size](https://docs.flutter.io/flutter/widgets/Icon/size.html)) dentro de layout
+primitivos. Además, se hace un uso intensivo de estructuras de datos inmutables
+incluyendo toda la jerarquía de clase de
+[Widget](https://docs.flutter.io/flutter/widgets/Widget-class.html), así como numerosas clases de apoyo tales como
+[Rect](https://docs.flutter.io/flutter/dart-ui/Rect-class.html) y
+[TextStyle](https://docs.flutter.io/flutter/painting/TextStyle-class.html). En una menor escala, La API
+[Iterable](https://docs.flutter.io/flutter/dart-core/Iterable-class.html) de Dart,
+que hace un uso intensivo del estilo funcional (map, reduce, where, etc.), es
+frecuentemente utilizado para procesar listas de valores en el framework.
+
+* Programación event-driven: las interacciones del usuario están representadas por objetos de eventos que se envían a callbacks registradas con controladores de eventos. Actualizaciones de pantalla son activados por un mecanismo de callback similar. La clase [Listenable](https://docs.flutter.io/flutter/foundation/Listenable-class.html), que se utiliza como base del sistema de animación, formatea un modelo de suscripción para eventos con múltiples oyentes.
+
+* Programación orientada a objetos basada en clases: la mayoría de las API del Framework se construyen usando clases con herencia. Usamos un enfoque por el cual definimos API de muy alto nivel en nuestras clases base, luego especialícelas iterativamente en subclases. Por ejemplo, nuestros objetos de renderizado tienen una clase base ([RenderObject](https://docs.flutter.io/flutter/rendering/RenderObject-class.html)) eso es agnóstico con respecto al sistema de coordenadas, y luego tenemos una subclase ([RenderBox](https://docs.flutter.io/flutter/rendering/RenderBox-class.html)) que introduce la opinión de que la geometría debe basarse en el cartesiano sistema de coordenadas (x/ancho y/ alto).
+
+* Programación orientada a objetos basada en prototipos: la clase [ScrollPhysics](https://docs.flutter.io/flutter/widgets/ScrollPhysics-class.html) se instancia para componer la física que se aplica al desplazamiento dinámicamente en tiempo de ejecución. Esto permite que el sistema componga, por ejemplo, paginación física con física específica de plataforma, sin que la plataforma tenga que ser seleccionado en tiempo de compilación.
```

Comment:
chains instances to compose the physics that apply to scrolling dynamically at runtime
encadena instancias para componer la fisica que aplica para hacer scroll dinámico en tiempo de ejecución.